hack/cluster-clean.sh

```diff
@@ -4,12 +4,28 @@ set -e
 
 source hack/common.sh
 
-echo "Deleting ocs-operator manifests"
+# Remove finalizers from all cephclusters, to not block the cleanup
+echo "Removing cephcluster finalizers"
+oc get cephcluster -n openshift-storage -o=custom-columns=NAME:.metadata.name,NAMESPACE:.metadata.namespace,FINALIZERS:.metadata.finalizers --no-headers | grep cephcluster.ceph.rook.io | while read p; do
+    arr=($p)
+    name="${arr[0]}"
+    namespace="${arr[1]}"
+    oc patch cephcluster $name -n $namespace --type=json -p '[{ "op": "remove", "path": "/metadata/finalizers" }]'
+done
+
+# delete ceph clusters and storage clusters
+echo "Deleting all storageclusters and cephclusters"
+oc -n openshift-storage delete storagecluster,cephcluster --all
+
 # even though we alter the deploy-with-olm.yaml in cluster-deploy,
 # the object names and namespaces remain the same, which allows us
 # to clean the cluster using the original manifest.
+echo "Deleting ocs-operator manifests"
 oc delete --ignore-not-found -f deploy/deploy-with-olm.yaml
 
+echo "Removing pods, pvcs, and pvs"
+oc -n openshift-storage delete pods,pvc,pv --all
```

Comment:
`oc delete --ignore-not-found -f deploy/deploy-with-olm.yaml` removes the `openshift-storage` namespace, right? Shouldn't this go before that?

Reply:
yeah, good point. I'm revising this...  it technically works as is, but not for obvious reasons.